File: js/Factorial/factorial.js
function factorial(n) {
    let factorial = 1;
    while(n > 1) {
        factorial *= n;
        n--;
    } 
    return factorial;
}

console.log(factorial(0));
console.log(factorial(1));
console.log(factorial(2));
console.log(factorial(3));
console.log(factorial(10));
console.log(factorial(20));
console.log(factorial(50));

File: java/tiny-problems/factorial.java
import java.util.*;

public class factorial {

    public static void main(String[] args) {
        int a = 0;
        int b = 1;
        int c;

        Scanner scanner = new Scanner(System.in);

        int number = scanner.nextInt();

        for (int i = 0; i < number; i++) {
            System.out.print(a + " ");

            c = a + b;
            a = b;
            b = c;
        }

    }
}